app/src/fdroid/java/com/eventyay/organizer/core/event/create/GooglePlacePicker.java

```diff
@@ -0,0 +1,35 @@
+package com.eventyay.organizer.core.event.create;
+
+import android.annotation.SuppressLint;
+import android.app.Activity;
+import android.content.Intent;
+
+public class GooglePlacePicker {
```

Comment:
Since this is also used in F-Droid, make this `PlacePicker` 

We'll be using OSM-Droid in future so it won't be GooglePlacePicker

Reply:
Actually PlacePicker won't be possible as PlacePicker has already been used in the code below.
```PlacePicker.IntentBuilder builder = new PlacePicker.IntentBuilder();```, so it throws error.
I am renaming it to ```LocationPicker```